weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -584,10 +584,10 @@ def update_FAST_model(self, fst_vt, inputs, discrete_inputs):
         fst_vt['ElastoDynTower']['TMassDen'] = inputs['mass_den'][twr_index:]
         fst_vt['ElastoDynTower']['TwFAStif'] = inputs['foreaft_stff'][twr_index:]
         fst_vt['ElastoDynTower']['TwSSStif'] = inputs['sideside_stff'][twr_index:]
-        fst_vt['ElastoDynTower']['TwFAM1Sh'] = inputs['fore_aft_modes'][0, :]  / sum(inputs['fore_aft_modes'][0, :])
-        fst_vt['ElastoDynTower']['TwFAM2Sh'] = inputs['fore_aft_modes'][1, :]  / sum(inputs['fore_aft_modes'][1, :])
-        fst_vt['ElastoDynTower']['TwSSM1Sh'] = inputs['side_side_modes'][0, :] / sum(inputs['side_side_modes'][0, :])
-        fst_vt['ElastoDynTower']['TwSSM2Sh'] = inputs['side_side_modes'][1, :] / sum(inputs['side_side_modes'][1, :])
+        fst_vt['ElastoDynTower']['TwFAM1Sh'] = np.abs(inputs['fore_aft_modes'][0, :])  / sum(np.abs(inputs['fore_aft_modes'][0, :]))
```

Comment:
Should this adjusted normalization happen in WISDEM so that the blade ElastoDyn benefits too?